### DISCUSSION TITLE
Rinkeby transaction indexing for hours

### DISCUSSION THREAD

**Discussion by Willfromtoronto** [smartcontractkit/full-blockchain-solidity-course-py#868] [Q&A] [answered]
Sometimes when I deploy a contract it takes hours (4 and counting) and sometimes it is almost immediate.  The transaction status is "indexing" and I cannot find the contract when searching on etherscan rinkeby.  How do I speed this up?

**Comment by cromewar** [comment, marked as answer]
Hello @Willfromtoronto  You can increase the gas price in order to get priority ;)
